src/backend/app/projects/project_schemas.py

```diff
@@ -365,6 +377,11 @@ async def all(
 
                 WHERE (p.author_id = COALESCE(%(user_id)s, p.author_id))
                 AND p.name ILIKE %(search)s
+                AND (
+                    %(user_id)s IS NOT NULL
+                    OR p.requires_approval_from_regulator = 'f'
```

Comment:
It's not mandatory, but it’s recommended to replace `OR` with `AND` for stricter filtering. This would ensure that all conditions must be met for the projects to be included in the list.